service/src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -379,4 +382,39 @@ public AzureResourceManager getResourceManager(AzureCloudContext azureCloudConte
         .withTenantId(azureCloudContext.getAzureTenantId())
         .withSubscription(azureCloudContext.getAzureSubscriptionId());
   }
+
+  private ComputeManager buildComputeManager(
+      AzureCloudContext azureCloudContext, AzureConfiguration azureConfig) {
+    TokenCredential azureCreds = getManagedAppCredentials(azureConfig);
+
+    AzureProfile azureProfile =
+        new AzureProfile(
+            azureCloudContext.getAzureTenantId(),
+            azureCloudContext.getAzureSubscriptionId(),
+            AzureEnvironment.AZURE);
+
+    // We must use FQDN because there are two `Defaults` symbols imported otherwise.
```

Comment:
Kinda unfortunate, maybe we can name the Azure class differently..

Reply:
Sounds like a good idea to me.